scripts/run_test_suite.sh

```diff
@@ -20,7 +20,9 @@ set -ex
 
 # Run php-cs-fixer.
 # We want to fail fast for coding standard violations.
-vendor/bin/php-cs-fixer fix --dry-run --diff .
+if [ -z "${SKIP_CS_CHECK}" ]; then
+    vendor/bin/php-cs-fixer fix --dry-run --diff .
```

Comment:
nit: we should remove the `.` when we upgrade `.php_cs` because then the filesystem is determined in the configuration file.